Makefile

```diff
@@ -130,6 +130,11 @@ spec: $(GENERATED_RPM_FILES)
 	
 test-rpmbuild: spec
 	${CONTAINER_RUNTIME} build -f images/rpmbuild/Containerfile .
+	${CONTAINER_RUNTIME} build -t test-rpmbuild-img -f images/rpmbuild/Containerfile .
+	${CONTAINER_RUNTIME} create --name test-rpmbuild test-rpmbuild-img
```

Comment:
I would go with `${CONTAINER_RUNTIME} build -v $(pwd)/images/rpmbuild/rpms/:/root/rpmbuild/:Z`, then you have `/root/rpmbuild/RPMS` readily available without going through hoops, and you also have `images/rpmbuild/rpms/BUILD/crc-2.12.0-4.11.18/release/crc-linux-amd64.tar.xz` which removes the need for an additional build of the crc binary.

Reply:
@cfergeau `-v` options for build doesn't mount host to container so this not going work. 

> I'm not sure the 2 ${CONTAINER_RUNTIME} build lines are intentional?

ah, L#132 should be removed.

Reply:
> @cfergeau -v options for build doesn't mount host to container so this not going work.

? I tested it before posting my comment and it worked as expected

Reply:
podman VS docker difference maybe?

Reply:
@cfergeau docker doesn't have this capability to volume mount during build, with latest version of docker it is possible but the using buildkit, but then the command for build change for docker and podman so I think better to have it this way. I can put that as comment or in commit message it that would make sense about why we are doing this way.  https://stackoverflow.com/questions/26050899/how-to-mount-host-volumes-into-docker-containers-in-dockerfile-during-build